academic-services/graph/network-similarity-databricks-gettopentities.md

```diff
@@ -0,0 +1,42 @@
+---
+title: NetworkSimilarity.getTopEntities Function
+description: NetworkSimilarity.getTopEntities Function
+services: microsoft-academic-services
+ms.topic: extra
+ms.service: microsoft-academic-services
+ms.date: 9/23/2020
+---
+# NetworkSimilarity.getTopEntities Function
+
+## getTopEntities(e, targetType, maxCount, minScore)
```

Comment:
again, entityId would probably be a better choice here for a public API.

Reply:
This is marked as resolved, but I don't see a code change, should it be 'won't fix'?

---
In reply to: [498410282](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/181#discussion_r498410282) [](ancestors = 498410282)